source/concepts.tex

```diff
@@ -29,6 +29,21 @@
 given equal inputs, the expression results in equal outputs. The inputs to an
 expression are the set of the expression's operands. The output of an expression
 is the expression's result and all operands modified by the expression.
+For the purposes of this subclause,
+the operands of an expression are the largest subexpressions that include only:
```

Comment:
Was it intentional that _operands_ is not formatted as a definition?

Reply:
Thinking about it now, we shouldn't make "operand" an italics definition here, because it means something different than the core-language "operand" of expressions. And since it's a local definition only, we're probably ok as-is.